packages/console/src/hooks/use-api.ts

```diff
@@ -56,8 +73,8 @@ export const useStaticApi = ({
           beforeError: hideErrorToast
             ? []
             : [
-                (error) => {
-                  void toastError(error.response);
+                async (error) => {
+                  await toastError(error.response);
```

Comment:
Why do we `await` here?

Reply:
Hold page render till confirm modal is addressed